content/docs/strict-mode.md

```diff
@@ -21,7 +21,7 @@ In the above example, strict mode checks will *not* be run against the `Header`
 * [Warning about deprecated findDOMNode usage](#warning-about-deprecated-finddomnode-usage)
 * [Detecting unexpected side effects](#detecting-unexpected-side-effects)
 * [Detecting legacy context API](#detecting-legacy-context-api)
-* [Detecting unsafe effects](#detecting-unsafe-effects)
+* [Ensuring reusable state](#ensuring-reusable-state)
```

Comment:
Drive-by fix. Currently the fragment link is broken: https://reactjs.org/docs/strict-mode.html#detecting-unsafe-effects